apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -163,6 +163,7 @@ export default function Success(props: PageProps) {
   const [calculatedDuration, setCalculatedDuration] = useState<number | undefined>(undefined);
   const [comment, setComment] = useState("");
   const parsedRating = rating ? parseInt(rating, 10) : 3;
+  const currentUserEmail = searchParams?.get("email") ?? session?.user?.email ?? undefined;
```

Comment:
when coming from the email cancel button then we don't have an `email` param but the `cancelledBy` param

Reply:
Should be both actually. When a booking is made, the booker is redirected to booking page where it also has email param. Forgot to include the cancelledBy for the email